src/common/extension-telemetry-events.ts

```diff
@@ -60,6 +60,7 @@ export const VISUAL_HELPER_TOGGLED: string = 'visualHelperToggled';
 export const ALL_RULES_EXPANDED: string = 'allRulesExpanded';
 export const ALL_RULES_COLLAPSED: string = 'allRulesCollapsed';
 export const RESCAN_VISUALIZATION: string = 'rescanVisualization';
+export const EXISTING_TAB_UPDATED: string = 'existingTabUpdated';
```

Comment:
The new event we are adding on this PR is about the url being changed on an existing tab but then name of the event looks more generic than that. Would something like `'existingTabUrlUpdated'` be better?